pkg/controllermanager/controller/plant/plant_test.go

```diff
@@ -172,5 +176,23 @@ var _ = Describe("Plant", func() {
 				Status: corev1.NodeStatus{Conditions: []corev1.NodeCondition{{Type: corev1.NodeReady, Status: corev1.ConditionFalse}}},
 			}, BeNil(), Not(BeTrue())),
 		)
+		DescribeTable("checkClusterNodes - returns correct condition on failure",
+			func(caseMatcher types.GomegaMatcher) {
+				var (
+					conditionEveryNodeReady = helper.InitCondition(gardencorev1alpha1.PlantEveryNodeReady)
+					discoveryMockclient     = mockdiscovery.NewMockDiscoveryInterface(ctrl)
```

Comment:
You could define `discoveryMockClient` on a context level and reset it with `mockdiscovery.NewMockDiscoveryInterface(ctrl)` in `BeforeEach()` instead of letting each test declare their own var.

Reply:
Thanks!